src/pybind/mgr/dashboard/frontend/src/app/core/navigation/about/about.component.html

```diff
@@ -1,26 +1,35 @@
+<div class="about-modal-pf">
 <div class="modal-header">
   <button type="button"
           class="close pull-right"
           aria-label="Close"
           (click)="modalRef.hide()">
-    <span aria-hidden="true">&times;</span>
+    <span class="pficon pficon-close"></span>
   </button>
 </div>
 <div class="modal-body">
-  <h1>ceph</h1>
-  <div class="product-versions">
-    <strong>Ceph version</strong>
-    {{ versionNumber }}
+  <h1>Red Hat Ceph Storage Dashboard</h1>
+  <div class="product-versions-pf">
+    <ul class="list-unstyled">
+      <li><strong>Version</strong> 3.2</li>
+      <li><strong>Ceph Manager</strong> {{ hostName }}</li>
+      <li><strong>User</strong> {{user}}</li>
+      <li><strong>User Role</strong> admin</li>
```

Comment:
Can't this be obtained from the localStorage or other service? In case it cannot, I think it's better not to put it, as non-admin users will see admin role here.

Reply:
ack, it can be obtained from users api